presto-main/src/test/java/com/facebook/presto/server/TestServer.java

```diff
@@ -103,6 +104,7 @@ public void testQuery()
                 .setHeader(PRESTO_SOURCE, "source")
                 .setHeader(PRESTO_CATALOG, "catalog")
                 .setHeader(PRESTO_SCHEMA, "schema")
+                .setHeader(PRESTO_ADDITIONAL_CONTEXT, "{\"field\":\"additionalContext\"}")
```

Comment:
Here is where this is actually useful as documentation. Use `"{\"additionalContext\":null}"`